app/src/main/java/com/babylon/wallet/android/presentation/ui/model/factors/FactorSourceInstanceCard.kt

```diff
@@ -5,14 +5,34 @@ import com.radixdlt.sargon.FactorSourceId
 import com.radixdlt.sargon.FactorSourceKind
 import com.radixdlt.sargon.Persona
 import kotlinx.collections.immutable.PersistentList
+import kotlinx.collections.immutable.persistentListOf
 
 data class FactorSourceInstanceCard(
-    val id: FactorSourceId.Hash,
```

Comment:
why this change? 👀

Reply:
Sargon works with `FactorSourceId` so it's easier if we use it as well to prevent unnecessary conversion operations.